### ISSUE TITLE
[Feature Request] Use adressbook in SMS sender. 

### ISSUE THREAD

**Issue by moteprime** [vikoadi/indicator-kdeconnect#50]
Thank you for this excellent indicator!

When sending a SMS from the indicator it's necessary to copy/paste from another application or manually type in the telephone number of the receiving telephone.
Being able to use gnome-contacts or maybe just remembering previous numbers i would be very helpful.

**Comment by JPFrancoia** [comment]
Yep, @moteprime is right, I think this is kind of vital. Also, in the sens SMS windows, there is a refresh button. Clinking it "synchronises the contacts". What does it actually do ?

**Comment by clawoflight** [comment]
*Some* other way of accessing contacts would be awesome. I don't use google contacts (sync with davical instead), so I cannot currently use this.
